content/altinn-studio/getting-started/app-dev-course/modul7/_index.nb.md

```diff
@@ -0,0 +1,138 @@
+1. Klikk på **Datamodell**.
+2. Klikk på **Legg til**.
+3. Klikk på **Ja/nei**.
+4. Endre navnet på feltet til ```KanBrukeSkjema```.
+5. Klikk **Generer modeller**.
+
+Datamodellen skal nå se slik ut:
+
+!["Datamodell med KanBrukeSkjema"](datamodell.png)
+
+### Legg til avkrysningsbokser
+Vi legger til avkrysningsbokser på infosiden så brukeren kan bekrefte at skjemaet er relevant for dem.
+1. Klikk på **Utforming**.
+2. Klikk på **infoside** og dra inn en komponent av typen **Radioknapper**.
+    !["Skjermbilde av radioknapper-komponenten satt inn i skjemabyggeren"](skjemabygger_med_radioknapper.png)
+3. Klikk på **Ledetekst** under **Tekst**-menyen til komponenten og kopier inn teksten:
+    ```Jeg møter kriteriene for å bruke appen.```
+4. Lag en ny kodeliste ved å klikke på **Sett opp egne alternativer** og **Legg til alternativ**.
+5. Klikk på **Radioknapp 1** og sett verdien til ```true``` og ledeteksten til ```Ja```.
+6. Klikk på **Legg til alternativ** igjen.
+7. Klikk på **Radioknapp 2** og sett verdien til ```false``` og ledeteksten til ```Nei```.
+   !["Radioknapper for bekrefte relevant app"](radioknapper_bekreftelse.png)
+8. Klikk på **Radioknapper** under **Datamodellknytninger**-menyen 
+9. Velg **KanBrukeSkjema** som datamodellknytning. 
+
+
+### Vis og skjul sider ved hjelp av logiske uttrykk
+Sidene **Side1** og **Arbeidsforhold** skal vises når brukeren har bekreftet at de oppfyller kriteriene for appen. Dette oppnås 
+ved å bruke logiske uttrykk, som styrer når en side skal vises basert på data fra brukeren. 
+1. Klikk på **Side1**.
+2. Klikk på **Valg for dynamikk**.
+3. Klikk på **Gjør om til logisk uttrykk**.
+4. Klikk på **Endre** for å åpne komponenten som brukes til å bygge logiske uttrykk.
+5. For den første operanden velg **Datamodell** som type. Deretter velger du datamodellfeltet **KanBrukeSkjema**.
+6. For operator velg **er ikke lik**.
+7. For den andre operanden velg **Sann/Usann** som type og sett verdien til **Sann**.
+
+!["Skjermbilde av dynamikkinnstillinger for Side1"](dynamikk_Side1.png)
+
+Gjenta prosessen for "Arbeidsforhold" ved å klikke på **Arbeidsforhold** i steg 1.
+
+For siden **ikke-for-deg** skal du bruke samme prosess, men med én viktig forskjell: I steg 7 skal du sette den andre operandens 
+verdi til **Usann** i stedet for **Sann**.
+
+## Vis stillingsutlysning for brukere med IKT-kompetanse
+Sogndal kommune ønsker å vise stillingsannonse når brukeren har markert at de har IKT-kompetanse.
+### Legg inn stillingsutlysningen
+Første steg er å legge inn stillingsannonsen i skjemaet.
+1. Klikk på **Utforming** og på siden for **Arbeidsforhold**.
+2. Dra inn en **Avsnitt**-komponent og legg den mellom komponentene **Bransje** og **Antall år i arbeidslivet**.
+   !["Skjermbilde av avsnittskomponenten satt inn i skjemabyggeren"](skjemabygger_med_avsnitt.png)
+3. Klikk på **Ledetekst** i **Tekst**-menyen for avsnittskomponenten, og kopier inn følgende tekst:
```

Comment:
```suggestion
3. Klikk på **Ledetekst** i **Tekst**-menyen for avsnittskomponenten, og kopier inn følgende tekst:
```Klikk på **Ledetekst** under **Tekst** for avsnittskomponenten, og kopier inn følgende tekst: